modules/delaunator-cpp/0.0.0-20181007-c1521f6/overlay/BUILD.bazel

```diff
@@ -0,0 +1,9 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+cc_library(
+    name = "delaunator-cpp",
+    hdrs = ["include/delaunator.hpp"],
+    includes = ["include/"],
```

Comment:
Does this need both `includes` and `strip_include_prefix`? I've also never seen the trailing slash in this context, but that doesn't mean I know it's wrong.

Reply:
Yes you are right. Does not make sense. Removed `strip_include_prefix`. Removed `/` from `includes`.